tests/e2e/scale/test_scale_small_file_workload.py

```diff
@@ -12,116 +13,125 @@
+        # Initial the Small Files workload, based on benchmark-operator
+        self.small_files = SmallFiles(self.es)
 
+        self.ceph = pod.get_ceph_tools_pod()
 
-def get_cephfs_data():
-    """
-    Look through ceph pods and find space usage on all cephfilesystem pods
+        # Collect the pulls usage befor the test is starting
+        self.orig_data = self.get_cephfs_data()
 
-    Returns:
-        Dictionary of byte usage, indexed by pod name.
-    """
-    ceph_pod = pod.get_ceph_tools_pod()
-    ceph_status = ceph_pod.exec_ceph_cmd(ceph_cmd="ceph df")
-    ret_value = {}
-    for pool in ceph_status["pools"]:
-        if "cephfilesystem" in pool["name"]:
-            ret_value[pool["name"]] = pool["stats"]["bytes_used"]
-    return ret_value
+    def teardown(self):
+        """
+        Teardown the test environment
 
+        """
+        self.small_files.cleanup()
+        self.es.cleanup()
 
-@pytest.fixture(scope="module")
-def scale_leaks(request):
-    """
-    For scale tests, get bytes used by cephfs at the start,
-    and compare with the cephfs usage at the finish.
-    """
+    def get_cephfs_data(self):
+        """
+        Look through ceph pods and find space usage on all ceph filesystem pods
 
-    def teardown():
+        Returns:
+            Dictionary of byte usage, indexed by pod name.
         """
-        After all the ripsaw creation and deltion runs have finished,
-        test to see if there was a leak in storage.
+        ceph_status = self.ceph.exec_ceph_cmd(ceph_cmd="ceph df")
+        ret_value = {}
+        for pool in ceph_status["pools"]:
+            if "cephfilesystem" in pool["name"]:
+                ret_value[pool["name"]] = pool["stats"]["bytes_used"]
+        return ret_value
+
+    def display_ceph_usage(self, msg, data):
         """
-        log.info("In scale_leaks teardown")
-        orig_data = cephfs_data.ceph_fs_use
-        now_data = get_cephfs_data()
-        still_going_down = True
-        while still_going_down:
-            log.info("Waiting for Ceph to finish cleaning up")
-            time.sleep(120)
-            new_data = get_cephfs_data()
-            still_going_down = False
-            for entry in new_data:
-                if new_data[entry] < now_data[entry]:
-                    still_going_down = True
-                    now_data[entry] = new_data[entry]
-        for entry in now_data:
-            # Leak indicated if over %50 more storage is used.
-            log.info(f"{entry} now uses {now_data[entry]} bytes")
-            log.info(f"{entry} originally used {orig_data[entry]} bytes")
-            ratio = now_data[entry] / orig_data[entry]
-            check = ratio < 1.50
-            errmsg = f"{entry} over 50% larger -- possible leak"
-            assert check, errmsg
+        Display the pool usage in a pretty way
 
-    request.addfinalizer(teardown)
+        Args:
+            msg (str): the message string to display with the values
```

Comment:
What kind of msg are we expecting here?

Reply:
The message string is a string which describe the data, i.e the stage of the data (before test / after creation / after deletion )

Reply:
ok, will be good to add that as a eg: just below the args.